exercises/conor_finn/ex02_oo_basics/src/circle.h

```diff
@@ -0,0 +1,16 @@
+#ifndef CIRCLE_H_
+#define CIRCLE_H_
+
+#include "shape.h"
+
+class Circle : public Shape {
+private:
+  const int radius;
```

Comment:
It is in the mantid standards to use an m_ prefix for non-static member variables i.e. `m_radius`